gpc-api-facade/src/main/java/uk/nhs/adaptors/gpc/amqp/task/TaskDefinition.java

```diff
@@ -0,0 +1,12 @@
+package uk.nhs.adaptors.gpc.amqp.task;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+
+@Getter
+@SuperBuilder
+@EqualsAndHashCode
+public class TaskDefinition {
```

Comment:
what about getting rid of that class completely? maybe we could add it when we need it?